PWGHF/TableProducer/HFBPlusToD0PiCandidateSelector.cxx

```diff
@@ -33,28 +34,31 @@ struct HfBplusTod0piCandidateSelector {
 
   Produces<aod::HFSelBPlusToD0PiCandidate> hfSelBPlusToD0PiCandidate;
 
-  Configurable<double> pTCandMin{"pTCandMin", 0., "Lower bound of candidate pT"};
-  Configurable<double> pTCandMax{"pTCandMax", 50., "Upper bound of candidate pT"};
+  Configurable<double> d_pTCandMin{"d_pTCandMin", 0., "Lower bound of candidate pT"};
+  Configurable<double> d_pTCandMax{"d_pTCandMax", 50., "Upper bound of candidate pT"};
```

Comment:
Why are you adding these prefixes?

Reply:
Just in order to be uniform with the other analysis tasks.
I noticed while modifying the .json file in the Run3ValidationFramework, and I realized after a while that names were different here with respect to other particle selectors tasks.
I can discard these changes if you think they are not necessary

Reply:
The prefixes are against the coding conventions. I will fix them soon elsewhere. So please do not add them back in your code.